### PR TITLE
[docs] Update FAQ description explaining what Expo is used for

### PR DESCRIPTION
**Pull request by ide**
# Why

Provides a stronger and broader description of Expo, especially for rendering this content in search results for people asking this question or learning about Expo.

# Test Plan

`yarn dev` => http://localhost:3002/introduction/faq/

### CODE REVIEW

**Review by Simek — APPROVED** [review]
LGTM, thank you for the update! 👍

**Review by amandeepmittal — APPROVED** [review]
👍

File: docs/pages/introduction/faq.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In addition to the questions below, see the [forums](https://forums.expo.dev/) f
 
 <Collapsible summary="What is Expo used for?">
 
-Expo is an [open-source npm package](https://github.com/expo/expo/) and framework for React Native projects. The `expo` package enables many important features for building and scaling an app like OTA updates, instantly sharing your app, and web support. Learn more about [what Expo offers](/introduction).
+Expo is an [open-source framework](https://github.com/expo/expo/) for apps that run natively on Android, iOS, and the web. Expo brings together the best of mobile and the web and enables many important features for building and scaling an app like live updates, instantly sharing your app, and web support. The `expo` npm package is compatible with all React Native projects. Learn more about [what Expo offers](/introduction).
```
Comment (EvanBacon) [inline docs/pages/introduction/faq.md:11]: This makes expo look like it's an alternative to React Native which leads to things like “expo vs React Native”.

Comment (ide) [reply docs/pages/introduction/faq.md:11]:
```suggestion
Expo is an [open-source framework](https://github.com/expo/expo/) for apps that run natively on Android, iOS, and the web. Expo brings together the best of mobile and the web and enables many important features for building and scaling an app like live updates, instantly sharing your app, and web support. The `expo` npm package also brings these features to any React Native project. Learn more about [what Expo offers](/introduction).
```

**Review by EvanBacon — APPROVED** [review]

**Pull request merged**